jetty-server/src/main/config/etc/jetty.xml

```diff
@@ -38,7 +38,7 @@
         <New class="org.eclipse.jetty.util.thread.ScheduledExecutorScheduler">
           <Arg name="name"><Property name="jetty.scheduler.name"/></Arg>
           <Arg name="daemon" type="boolean"><Property name="jetty.scheduler.daemon" default="false" /></Arg>
-          <Arg name="threads" type="int"><Property name="jetty.scheduler.threads" default="-1" /></Arg>
+          <Arg name="threads" type="int"><Property name="jetty.scheduler.threads" default="1" /></Arg>
```

Comment:
Is this a purposeful change or a mistake??

Reply:
Note sure why @sbordet changed it?   Even if there is a reason, this is the wrong PR to do it in. I will revert